src/Components/Home/ListItem.js

```diff
@@ -4,6 +4,10 @@ import { Card, Col, Row, Button, Badge } from 'react-bootstrap'
 function ListItem(prop) {
     const { id, name, daysLeft, likeCount } = prop.product;
 
+    const handleEditClick = () => {
+        prop.changePage(prop.pages.EDIT_PRODUCT)
```

Comment:
We could import all possible pages instead of passing using props:
`import pages from './pages'`
then `prop.changePage(pages.EDIT_PRODUCT)`
But in my opinion, you can leave like that, no problem just a different approach.

Reply:
I prefer your suggestion to drilling the pages prop through every component.